04-o-processor-spark/src/main/scala/com/fortyseven/processor/spark/SparkProcessor.scala

```diff
@@ -30,8 +30,8 @@ class SparkProcessor extends SparkProcessorAPI:
   private def runWithConfiguration(sparkProcessorConfiguration: SparkProcessorConfigurationI): Unit =
 
     val sparkConf: SparkConf = new SparkConf()
-      .setAppName(sparkProcessorConfiguration.app.appName)
-      .setMaster(sparkProcessorConfiguration.app.masterUrl)
+      .setAppName(sparkProcessorConfiguration.app.appName.toString)
+      .setMaster(sparkProcessorConfiguration.app.masterUrl.toString)
```

Comment:
`toString` can lead to some issues, as sometimes it is implemented with the hashcode or other stuff. Based on the specification, `toString` means a readable object string and shouldn't be treated as a value. Does that type have something like `.value` available?

Reply:
I have been iterating on this library at the client and `.toString` is also not recommended. There is a better use of the library. I will probably update this PR with the last team agreement. Soon!